testsuite/features/profiles/mgr.suse.de/Kiwi/POS_Image-JeOS6_32/config.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
```

Comment:
Same suggestions:

* invert `Kiwi` and location parts, keeping current hierarchy (again, no strong feelings...)
* make `mgr.suse.de` something less implementation-dependant, for example `internal_nue`

Reply:
done